### PR TITLE
[DE-665]: `GET /_api/transaction`

### PR DESCRIPTION
**Pull request by aMahanna**
- Introduces `list_transactions()` to list the running stream transactions

### CODE REVIEW

File: tests/test_transaction.py [hunk 1]
```diff
@@ -42,6 +42,12 @@ def test_transaction_execute_raw(db, col, docs):
     assert err.value.error_code == 10
 
 
+def test_transaction_list(db, col, docs):
+    no_transactions = db.list_transactions()
+    assert no_transactions == []
+    # TODO: Add proper transaction test here
```
Comment (aMahanna) [inline tests/test_transaction.py:48]: todo

File: tests/test_transaction.py [hunk 2]
```diff
@@ -42,6 +42,12 @@ def test_transaction_execute_raw(db, col, docs):
     assert err.value.error_code == 10
 
 
+def test_transaction_list(db, col, docs):
+    transactions = db.list_transactions()
+    assert isinstance(transactions, list)
+    # TODO: Add proper transaction test here
```
Comment (aMahanna) [inline tests/test_transaction.py:48]: todo

Comment (aMahanna) [reply tests/test_transaction.py]: added in [3d6c241](https://github.com/ArangoDB-Community/python-arango/pull/283/commits/3d6c24117f51bf59c68b5c7aeea6f32ad018365a)

**Review by apetenchea — APPROVED** [review]
LGTM

**Pull request merged**